### ISSUE TITLE
[Request] Missing sync/history get last watched ?

### ISSUE THREAD

**Issue by mmarquezs** [fuzeman/trakt.py#49]
Sorry for the horrible title I didn't know how to phrase it . Basically unless I missed something currently with this library it's impossible to get the last watched history ( http://docs.trakt.apiary.io/#reference/sync/get-history/get-watched-history ).

**Comment by fuzeman** [comment]
I've implemented support for history retrieval on the [develop](https://github.com/fuzeman/trakt.py/archive/develop.zip) branch *(see below for some examples)*. Will try get this released onto PyPI later this week after some more testing. :smiley:

----------------

Simple calls:
```python
# Retrieve one page of history records
items = list(Trakt['sync/history'].get(per_page=10))

# Iterate over one page of history records
for item in Trakt['sync/history'].get(per_page=10):
    ...

# Iterate over your entire history
for item in Trakt['sync/history'].get(pagination=True, per_page=100):
    ...
```

Media filtering:
```python
# Iterate over one page of movie history records
for item in Trakt['sync/history'].movies(per_page=10):
    ...

# Iterate over one page of show history records
for item in Trakt['sync/history'].shows(per_page=10):
    ...
```

Retrieve history for specific show or movie:
```python
# Iterate over history records for "Europa Report"
for item in Trakt['sync/history'].movies('110966', pagination=True, per_page=25):
    ...

# Iterate over history records for "Family Guy"
for item in Trakt['sync/history'].shows('1425', pagination=True, per_page=25):
    ...
```

Retrieve history records for a specific time span:
```python
# Iterate over history records since 2016-01-01
for item in Trakt['sync/history'].get(pagination=True, per_page=100, start_at=datetime(2016, 1, 1)):
    ...
```

**Comment by fuzeman** [comment]
Released in [v2.9.0](https://github.com/fuzeman/trakt.py/releases/tag/2.9.0)
